content/types/_rest_client_options.textile

```diff
@@ -1,3 +1,3 @@
-- <span lang="default">queryTime</span><span lang="python">query_time</span><span lang="ruby">:query_time</span><span lang="csharp">QueryTime</span> := _false_ If true, the library will query the Ably servers for the current time instead of relying on a locally-available time of day. Knowing the time accurately is needed to create valid signed token requests, so this option is useful for library instances on auth servers if the server clock is insufficiently accurate. The server is queried only once per client library instance, and used to calculate and store the offset from the local clock.<br>__Type: @Boolean@__
+- <span lang="default">queryTime</span><span lang="python">query_time</span><span lang="ruby">:query_time</span><span lang="csharp">QueryTime</span> := _false_ If true, the library will query the Ably servers for the current time instead of relying on a locally-available time of day. Knowing the time accurately is needed to create valid signed Ably TokenRequests, so this option is useful for library instances on auth servers if the server clock is insufficiently accurate. The server is queried only once per client library instance, and used to calculate and store the offset from the local clock.<br>__Type: @Boolean@__
 
-- <span lang="default">defaultTokenParams</span><span lang="python">default_token_params</span><span lang="ruby">:default_token_params</span><span lang="csharp">DefaultTokenParams</span> := When a "TokenParams":/rest/types#token-params object is provided, it will override the client library defaults when issuing new tokens or token requests<br>__Type: "@TokenParams@":/rest/types#token-params__
+- <span lang="default">defaultTokenParams</span><span lang="python">default_token_params</span><span lang="ruby">:default_token_params</span><span lang="csharp">DefaultTokenParams</span> := When a "TokenParams":/rest/types#token-params object is provided, it will override the client library defaults when issuing new Ably Tokens or Ably TokenRequests<br>__Type: "@TokenParams@":/rest/types#token-params__
```

Comment:
I think generally where we refer to Ably Tokens and Ably TokenRequests, we should take that opportunity to link to suitable documentation.  I don't think we should expect developers to know where to find information on  terms / keywords / acronyms etc.